src/09_dictionaries.py

```diff
@@ -36,12 +36,38 @@
 # Add a new waypoint to the list
 # YOUR CODE HERE
 
+# 1st option
+waypoints.append({"lat": 42, "lon": -124, "name": "a fourth place"})
+print("1st waypoints: ", waypoints)
+
+# 2nd option
+fifth_place = {
+    "lat": 45,
+    "lon": -125,
+    "name": "a fifth place"
+}
+waypoints.append(fifth_place)
+print("2nd waypoints: ", waypoints)
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # Note: It's okay to access the dictionary using bracket notation on the
 # waypoints list.
-
 # YOUR CODE HERE
 
+# 1st option
+for waypoint in waypoints:
+    if waypoint["name"] == "a place":
+        waypoint["lon"] = -130
+        waypoint["name"] = "not a real place"
+print("3rd waypoints: ", waypoints)
+
+# 2nd option
+waypoints[0]["name"] = "a place"
+waypoints[0]["lon"] = -121
```

Comment:
Since the `waypoints` list is defined within the scope and we know that the `name` with "a place" is the first element, the 2nd option is a cleaner way to go.